barretenberg/cpp/src/barretenberg/common/thread.hpp

```diff
@@ -89,4 +89,4 @@ inline void run_loop_in_parallel_if_effective_with_index(size_t num_points,
                                                group_element_doublings_per_iteration,
                                                scalar_multiplications_per_iteration,
                                                sequential_copy_ops_per_iteration);
-}
```

Comment:
@catmcgee can you fix this line? for some reason I can't get my formatter to

Reply:
fixed